docs/source/getting-started/content/proc_accessing-the-openshift-cluster.adoc

```diff
@@ -30,14 +29,18 @@ See <<troubleshooting-codeready-containers_{context}>> if you cannot access the
 
 * To access the OpenShift cluster via the [command]`oc` command, follow these steps:
 
-// TODO: Use attributes for the path here. Avoids manually maintaining it.
-  . Export the Kubernetes configuration for the {prod} OpenShift cluster:
+  . Run the [command]`{bin} oc-env` command to print the command needed to add the cached [command]`oc` binary to your `_PATH_`:
 +
 [subs="+quotes,attributes"]
 ----
-$ export KUBECONFIG="$HOME/.crc/cache/crc_libvirt_v4.1.0.rc0/kubeconfig"
+$ crc oc-env
+export PATH="/home/user/.crc/bin:$PATH"
+# Run this command to configure your shell:
+# eval $(crc oc-env)
```

Comment:
this would only apply to a shell on Linux or Un*x-like environments ;-)

it would almost be more appropriate to just say "Run `oc-env` and follow the instructions as given"

Reply:
Sure, makes sense.